packages/video_player/video_player_web/lib/src/video_player.dart

```diff
@@ -58,7 +58,9 @@ class VideoPlayer {
   /// This method sets the required DOM attributes so videos can [play] programmatically,
   /// and attaches listeners to the internal events from the [html.VideoElement]
   /// to react to them / expose them through the [VideoPlayer.events] stream.
-  void initialize() {
+  void initialize({
+    String? src,
```

Comment:
This could use some discussion in the documentation comment.

Reply:
Agreed, will update docs really quickly.